source/includes/js/_identify.md

```diff
@@ -82,9 +82,6 @@ The following are treated as special properties. All other data passed in will b
   success: true,
   visitor_uuid: "f627ee608adb01315d1022000ab2058a",
   anonymous: false,
-  email: "john@acme.com",
-  custom_fields: { "name": "John" },
-  tags: ["Customer"],
   lead_score: 35
```

Comment:
this is gone too:

```
    @payload = {
      success: true,
      visitor_uuid: @visitor_uuid,
      anonymous: @identity.anonymous?
    }
```

only those remain.

Reply:
Sorry it's not clear - what's else did we remove, @andrewstucki?

Reply:
sorry, `lead_score` shouldn't be on there either

Reply:
Ah! Thanks.